sql/dql/basic-queries/order-by-clause.md

```diff
@@ -41,117 +41,104 @@ aspects:
 ---
 ## Content
 
-`ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. It comes after `GROUP BY` clause.
+The `ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. This is also more commonly known as **sorting** data.
+
+Consider the following `employees` table:
+
+| id  | first_name | last_name | email                   |
+| --- | ---------- | --------- | ----------------------- |
+| 1   | Karlene    | Pantone   | kpantone0@topsy.com     |
+| 2   | Donnamarie | Beeho     | dbeeho1@state.tx.us     |
+| 3   | Tony       | Isworth   | tisworth2@bloglovin.com |
+| 4   | Esta       | Warrier   | ewarrier3@berkeley.edu  |
+| 5   | Immanuel   | Kippen    | ikippen4@biglobe.ne.jp  |
+| 6   | Regina     | Pantone   | regone@stratos.org      |
+
+Right now, the table is sorted in ascending order by the `id` column, but what if you need the results of your query to be sorted in ascending order by the `last_name` column? To achieve this you would use the following syntax: 
```

Comment:
I think this question can be misleading - "right now the table is sorted in ascending order" ... this is not always the case and might prompt people to think otherwise 

I think it's safe to skip this bit and just say "look at this table...say we want to sort the results through a query alphabetically by their last name"

Reply:
Changed the entire paragraph to:

Let's say that through a query we want our results sorted alphabetically by their `last_name`. To achieve this you would use the following syntax: